test/storage/varlen_pool_test.cpp

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Terrier
+//
+// varlen_pool_test.cpp
+//
+// Identification: test/storage/varlen_pool_test.cpp
+//
+// Copyright (c) 2015-18, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <limits.h>
+#include <pthread.h>
+
+#include "storage/varlen_pool.h"
+#include "gtest/gtest.h"
+
+namespace terrier {
+class VarlenPoolTests : public ::testing::Test {};
+
+// disable unused const variable warning for clang
+#ifdef __APPLE__
```

Comment:
Why do you need this? What unused const variable is there that you cannot get rid of?